tests/customerSite/cart/updateCart/cartUpdatedAfterClickingPlusButton.spec.js

```
@@ -1,52 +1,55 @@
-    COFFEE_NAMES.espresso,
-    oneEspressoPrice,
-  );
-
-  await cartPage.clickCoffeeListItemAddOneButton(COFFEE_NAMES.espresso);
-
-  await cartPage.assertCoffeeTotalCostContainsCorrectText(
-    COFFEE_NAMES.espresso,
-    twoEspressoPrice,
-  );
-  await cartPage.assertCoffeeTotalCostContainsCorrectText(
-    COFFEE_NAMES.cappuccino,
-    oneCappuccinoPrice,
-  );
-
-  await cartPage.clickCoffeeListItemAddOneButton(COFFEE_NAMES.cappuccino);
-
-  await cartPage.assertCoffeeTotalCostContainsCorrectText(
-    COFFEE_NAMES.cappuccino,
-    twoCappuccinoPrice,
-  );
-  await cartPage.assertCoffeeTotalCostContainsCorrectText(
-    COFFEE_NAMES.espresso,
-    twoEspressoPrice,
-  );
-
-  await cartPage.assertTotalCheckoutContainsValue(totalPrice);
+test.describe('Cart > Update with adding', () => {
+  // Shared Allure labels for this suite/file
+  test.beforeEach(async () => {
+    await allure.parentSuite('Customer site');
+    await allure.suite('Cart');
+    await allure.subSuite('Cart update with adding');
+    await allure.epic('CoffeeCart Customer site');
+    await allure.feature('Cart');
+    await allure.severity('critical');
+  });
+
+  test('Cart updated correctly after clicking plus for drinks', async ({ cartPage, menuPage }) => {
+    await allure.story('User can add a coffee cup to the cart');
+
+    const oneCap = priceFormatStr(COFFEE_PRICES.cappuccino);
+    const twoCap = priceFormatStr(COFFEE_PRICES.cappuccino * 2);
+    const oneEsp = priceFormatStr(COFFEE_PRICES.espresso);
+    const twoEsp = priceFormatStr(COFFEE_PRICES.espresso * 2);
+    const totalNum = COFFEE_PRICES.cappuccino * 2 + COFFEE_PRICES.espresso * 2;
+    const total = priceFormatStr(totalNum);
+
+    await allure.parameter('cappuccino(1x)', oneCap);
+    await allure.parameter('cappuccino(2x)', twoCap);
+    await allure.parameter('espresso(1x)', oneEsp);
+    await allure.parameter('espresso(2x)', twoEsp);
+    await allure.parameter('expected total', total);
```

Inside the test you call `await allure.story(...)` and `await allure.parameter(...)`. Same issue: prefer `testInfo.allure.story(...)` and `testInfo.allure.parameter(...)` by receiving `testInfo` in the test function (`async ({ cartPage, menuPage }, testInfo) => { ... }`). This ensures the labels/parameters are attached to the current test's Allure lifecycle. Also failing fast when `testInfo.allure` is undefined will surface reporter misconfiguration.